zos_concepts/gdg_datasets/copy_edit_fetch/site.yml

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# © Copyright IBM Corporation 2024
+###############################################################################
+
+###############################################################################
+# This playbook (`site.yml`) is the master playbook for this project. Based on
+# the playbooks requirments, `site.yml` playbook will evalute the requirements
+# and report when the minimum requirements are not configured and fail playbook
+# execution.
+#
+# Upon successful dependency analyais, the z/OS playbook will be executed.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories site.yml
+#  ansible-playbook -i inventories site.yml -v
+###############################################################################
+
+
+---
+- hosts: localhost
+
+  tasks:
+    - name: Parse the registry for collection entries
+      include_vars: runtime-requirements.yml
+
+    - name: Display the registered collections and properites in `registry.yml`.
+      include_role:
+        name: requirements-check
+
+- name: Executing the z/OS Ansible playbook
+  import_playbook: copy_edit_fetch.yml
```

Comment:
The playbook is `create_copy_submit.yml` not `copy_edit_fetch.yml`

Reply:
At this point it is `copy_edit_fetch.yml` so is correct.